packages/protobuf/src/registry.ts

```diff
@@ -38,9 +36,6 @@ export const typeRegistry: IMessageTypeRegistry = createRegistry(
   ClientState,
   Header,
 
-  // gen/ibc/core/channel/v1/tx_pb
-  MsgRecvPacket,
-
```

Comment:
Not necessary to list specifically in the registry as it exists in `IbcChannelMsgService` already